README.md

```diff
@@ -33,6 +32,7 @@ Related [Project-OSRM](https://github.com/Project-OSRM) repositories:
 
 ## Contact
 
+- Discord: [https://discord.gg/es9CdcCXcb](join)
 - IRC: `irc.oftc.net`, channel: `#osrm` ([Webchat](https://webchat.oftc.net))
```

Comment:
Do we still need links to IRC and Mailinglist here? They seem to be completely dead...

Reply:
There are some useful historical discussions in the mailing list, so good to keep as a reference. Not so sure about IRC.